### PR TITLE
Documentation/123-improve-code-examples

### LINKED ISSUE #123: Improve documentation
* Add/fix links
* Code examples in the documentation could be improved:
  * remove `final` declarations
  * remove trailing `//` comments

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #123

### CODE REVIEW

File: doc/user_guide/rendering.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
 
 ## Creating the Renderer With Default Settings
 
-To render an SQL statement, you need to create an instance of the SQl  Statement Renderer (for example `CreateTableRenderer`) using the `create()` method of
-the Renderer class.
+To render an SQL statement, you need to create an instance of the SQl Statement Renderer (for example `CreateTableRenderer`) using the `create()` method of
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/rendering.md:5]:
```suggestion
To render an SQL statement, you need to create an instance of the SQL Statement Renderer (for example `CreateTableRenderer`) using the `create()` method of
```

File: doc/user_guide/rendering.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
+the Renderer class:
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/rendering.md:6]: We usually don't use hard wraps in the docs, so if you see them you can fix them inlining

File: doc/user_guide/statements/create_schema.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # CREATE SCHEMA
 
 The `CreateSchema` class of the SQL Statement Builder provides an entry 
-point to defining a CREATE SCHEMA SQL statement.
+point to defining a [`CREATE SCHEMA`](https://docs.exasol.com/sql/create_schema.htm) SQL statement.
```
Comment (AnastasiiaSergienko) [inline doc/user_guide/statements/create_schema.md:4]: This also should be inlined

**Review by AnastasiiaSergienko — COMMENTED** [review]
A few small comments

File: doc/user_guide/rendering.md (hunk 2 above)
Comment (kaklakariada) [reply doc/user_guide/rendering.md:6]: Thanks! I was not sure, so I left them :)

**Review by AnastasiiaSergienko — APPROVED** [review]
:+1:

**Pull request merged**